RFC/src/RFC-0100_BaseLayer.md

```diff
@@ -6,11 +6,11 @@
 
 **Maintainer(s)**: [Cayle Sharrock](https://github.com/CjS77)
 
-# License
+# Licence
 
-[ The 3-Clause BSD License](https://opensource.org/licenses/BSD-3-Clause).
+[The 3-Clause BSD Licence](https://opensource.org/licenses/BSD-3-Clause).
 
-Copyright 2018 The Tari Development Community
+Copyright <YEAR> <COPYRIGHT HOLDER | The Tari Development Community>
```

Comment:
Please replace `Copyright <YEAR> <COPYRIGHT HOLDER | The Tari Development Community>` 
with `Copyright 2018 The Tari Development Community`

The original text was correct. The date will change dependent on the year, so most RFCs will have 
`Copyright 2019 The Tari Development Community`